app/policies/school_policy.rb

```diff
@@ -2,6 +2,9 @@ class SchoolPolicy < ApplicationPolicy
   def show?
     return false if user.blank?
 
+    # record should belong to current school
+    return false unless record == current_school
```

Comment:
The `record` in question here is always the current school. This is the `SchoolPolicy` after all. I don't think this serves any purpose. 🤔